draft-ietf-httpbis-digest-headers.md

```diff
@@ -1030,7 +1057,18 @@ before handing off to say the HTML parser, video player etc.
 
 Even a simple mechanism for end-to-end validation is thus valuable.
 
-## Usage in Signatures
+## Digest and Content-Location in responses {#digest-and-content-location}
+
+When a state-changing method returns the `Content-Location` header field,
+the enclosed representation refers to the resource identified by its value
+and `Digest` is computed accordingly.
+
+It's important to note though that this claim cannot be trusted
+unless it can be verified by other means (see Section 3.1.4.1 and 3.1.4.2 of [RFC7231]).
+A malicious intermediary may add a `Content-Location` header field referencing an external resource
+thus asserting that the enclosed representation and its `Digest` refers to it.
```

Comment:
I'm not 100% sure what "external resource" means in this consideration, nor what the malicious intermediary can achieve by this type of attack. 

It might help to be a little more specific here, by my understanding we might want to say something like the candidate text - but maybe I'm completely off?

```
A malicious intermediary may add a `Content-Location` header field containing an absolute URL that references
a resource on a different host than the original request. This would assert that the enclosed representation and its `Digest` relate to the third-party resource. The effect on a client attempting to validate the `Digest` is [insert here].
```

Reply:
@LPardue wdyt about removing lines 1066-1069? They are a repetition of the RFC 7231 and 
it's probably better to remove them.

Reply:
Let's drop it and merge the PR